tests/lib/bash/test_header

```diff
@@ -75,6 +75,12 @@
 #         echo "1..0 # SKIP SKIP_REASON" and exit.
 #     ssh_install_cylc HOST
 #         Install cylc on a remote host.
+#     create_test_globalrc [PRE [POST]]
+#         Create a new global config file $PWD/conf from site global-tests.rc
+#         with PRE and POST pre- and ap-pended (PRE for top level items with no
+#         section heading). For multi-line strings in bash, use:
+#           $"the quick
```

Comment:
OK, looks like I am wrong.  The `$"abc\ndef"` form is only needed to embed a newline as `\n`.